src/SpectatorView.Unity/Assets/SpatialAlignment/README.md

```diff
@@ -10,25 +14,77 @@ Not all spatial alignment strategies support all platforms. See the chart below
+
+### Registration & Configuration
+
+Both of these aspects are required to enable a method to be used for localization, however, they are slighly different. `SpectatorView` comes pre-registered with several `SpatialLocalizers` that can be found on `SpectatorView\Prefabs\SpectatorView.SpatialCoordinateLocalizers.prefab`:
+
+- Azure Spatial Anchors (`SpatialAnchorsLocalizer`) localizer will rely on the hosting (User) device to create the common ISpatialCoordianate to be used by all ASA spectating devices connecting
+- Physical Marker localizers will search for some physical marker in the world
+- Marker Visual localizer pairs will display a marker on the screen of a mobile device to be discovered by the other device.
+
+Some of these localization methods require settings, which are set through a `SpatialLocalizerInitializer`, two can be found on that prefab for the QR and ArUco visual localizers. Additional settings must be added manually by the consuming application onto `Generated.StateSynchronization.AssetCaches\Resources\SpectatorViewSettings.prefab` which is created by invoking the `Spectator View > Edit Settings` menu item, see [Spatial Alignment Dependencies](../../../../doc/SpectatorView.Setup.md##%20Spatial%20Localizer%20Dependencies).
+
+### Selection of Localization Method
+
+When the application starts and `SpectatorView` is initialized, configured localizers are checked for whether they are supported in the current application on the current device and if they are, they are registered with `SpectatorView`. Afterwards, the process is as follows:
 
-1. On startup, SpatialLocalizers declared in the scene register themselves with the SpatialCoordinateSystemManager. Each SpatialLocalizer enables a different spatial alignment strategy and has a unique id so that it can be invoked through local or remote spatial alignment requests.
+1. `SpatialCoordinateSystemManager` listens for incoming/outgoing network connections, creating a `SpatialCoordinateSystemParticipant` for each connection.
+2. `SpectatorView` on the spectating device listens for the creationsof these participants, and queries for support localizers of the participant.
+3. Then, based on it's own configured priotized list of `SpatailLocalizationInitializers` and the supported list returned by the hosting (User) device, it identifies the best localization method to use.
 
-2. The SpatialCoordinateSystemManager then listens for network connections (This is facilitated through state synchronization's ICommandRegistry). For each connection, the SpatialCoordinateSystemManager creates a SpatialCoordinateSystemParticipant, which is responsible for monitoring and reporting coordinate state across devices. The SpatialCoordinateSystemParticipant also caches the associated network socket to allow for sending messages across devices during the spatial alignment process.
+> Note: Best is determined as the lowest index of the supported localizers in it's configured list.
```

Comment:
nit: This may move around. Please change "Best" to "The best localization method" or something signaling what the item is the best at.